src/pages/index.tsx

```diff
@@ -55,6 +57,20 @@ const Pages = () => {
                     <Route path={"/admin/*"} Component={AdminRouter} />
                     <Route path={"/:orgUrl/*"} Component={OrgRouter} />
                 </Routes>
+                {showDesignBanner && (
+                    <div
+                        style={{ zIndex: 51 }}
+                        className="fixed flex bottom-0 w-full h-12 border-t border-stone-700 bg-stone-900/85 backdrop-blur-xl items-center justify-center"
+                    >
+                        <Typography className="text-center">Epsilon is undergoing redesign until March 2025. Please excuse our appearance.</Typography>

```

Comment:
Instead of "until March 2025" perhaps we keep it more vague at a "Epsilon is currently undergoing a design overhaul. Please excuse our appearance while we work to improve the Epsilon interface."

Reply:
OK